File: FE.Core.FW/src/apis/models/DecisionBlacklistModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type DecisionBlacklistModel = {
  id?: string;
  blacklistId?: string;
  decisionNumber?: string | null;
  decisionDate?: string | null;
  startDate?: string | null;
  endDate?: string | null;
  reason?: string | null;
  formProcess?: number | null;
  examIdBan?: string | null;
  dateApprove?: string | null;
  isInclude?: boolean;
  typeOfDecision?: number;
  createdBy?: string | null;
  approveBy?: string | null;
  note?: string | null;
  status?: number | null;
  filePath?: string | null;
  createdOnDate?: string | null;
  fileFile?: Blob | null;
};

File: BE.Core.FW/Backend/Business/DecisionBlacklist/DecisionBlacklistModel.cs
using System.ComponentModel.DataAnnotations;

namespace Backend.Business.DecisionBlacklist
{
    public class DecisionBlacklistModel
    {
        public Guid Id { get; set; }
        public Guid BlacklistId { get; set; }
        public string? DecisionNumber { get; set; }
        public DateTime? DecisionDate { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public Guid? Reason { get; set; }
        public int? FormProcess { get; set; }
        public string? ExamIdBan { get; set; }
        public DateTime? DateApprove { get; set; }
        public string? CreatedBy { get; set; }
        public string? ApproveBy { get; set; }
        public string? Note { get; set; } = string.Empty;
        public int? Status { get; set; }
        public string? FilePath { get; set; }
        public string? CreatedOnDate { get; set; }
        public IFormFile? FileFile { get; set; }
    }
}
